src/commands/new.ts

```diff
@@ -366,7 +370,7 @@ module.exports = {
           format: prettyPrompt.format.boolean,
           prefix,
         }))
-        stateMgmt = includeMSTResponse.includeMST
+        stateMgmt = includeMSTResponse.includeMST ? "mst" : "none"
```

Comment:
Was this coming being set as the wrong type of data? According to TS this should always be `"mst" | "none"` already. if that wasn't the case do we have a type error in prompt.ask?

Reply:
Yeah probably, it was being set to a boolean value